plugins/node/opentelemetry-instrumentation-redis/src/utils.ts

```diff
@@ -102,7 +105,7 @@ export const getTracedInternalSendCommand = (
       {
         kind: SpanKind.CLIENT,
         attributes: {
-          [SemanticAttributes.DB_SYSTEM]: RedisInstrumentation.COMPONENT,
+          [SemanticAttributes.DB_SYSTEM]: DbSystemValues.REDIS,
```

Comment:
is `RedisInstrumentation.COMPONENT` still used anywhere?  if not it should be removed.

Reply:
this one is still used